plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveConfig.java

```diff
@@ -101,7 +101,8 @@ public void testDefaults()
                 .setDynamicFilteringProbeBlockingTimeout(new Duration(0, TimeUnit.MINUTES))
                 .setTimestampPrecision(HiveTimestampPrecision.DEFAULT_PRECISION)
                 .setOptimizeSymlinkListing(true)
-                .setLegacyHiveViewTranslation(false));
+                .setLegacyHiveViewTranslation(false)
+                .setReadColumnIndexFilter(true));
```

Comment:
You get a smaller diff if you set this _above_ the existing last line, so the punctuation that's already there doesn't have to be moved. (This applies in a few other places as well.)